Modify API docs in process.md (#35590)
nodejs/node: doc/api/process.md

* {string}

The `process.execPath` property returns the absolute pathname of the executable

> that started the Node.js process. Symbolic links if any, will be resolved

Reviewer: ```suggestion
that started the Node.js process. Symbolic links, if any, will be resolved
```

Reviewer 2: In addition to the comma, I'd prefer that we keep things in present tense if there's a choice:

```suggestion
that started the Node.js process. Symbolic links, if any, are resolved
```

Author: @addaleax and @Trott - thanks for the review, I have followed your suggestion. Please have a look.

> that started the Node.js process. Symbolic links if any, will be resolved
> in the pathname.

Reviewer 2: ```suggestion
that started the Node.js process. Symbolic links, if any, are resolved.
```

Author: @Trott -Thanks. I have followed your suggestion. Please have a look.
